src/lib/translate/index.js

```diff
@@ -47,7 +47,8 @@ function dereferenceNativeObject (nativeObject, movieClipRefs = [], shapeRefs =
                     bn: getBlockNameVar(resolvedValue.id, blockNameMappings),
                     gn: resolvedValue.id,
                     a: resolvedValue.data.constructorArgs,
-                    t: resolvedValue.data.transform
+                    t: resolvedValue.data.transform,
+                    ...(resolvedValue.data.off && {off: resolvedValue.data.off})
```

Comment:
These expressions can be simplified to `(resolvedValue.data.off === true)` which will explicitly set the value to true or false.

Reply:
Updated to `off: resolvedTarget.data.off === true`. Is this what you intended?